tofhir-server/src/test/scala/io/tofhir/server/project/SchemaEndpointTest.scala

```diff
@@ -76,6 +78,8 @@ class SchemaEndpointTest extends BaseEndpointTest {
   // fifth schema with the same url as the second schema, to be rejected
   val schema5: SchemaDefinition = SchemaDefinition(url = "https://example.com/fhir/StructureDefinition/schema2", `type` = "ty5", name = "name5", rootDefinition = None, fieldDefinitions = None)
 
+  // blood pressure schema for import test (Does not include root and field definitions)
+  val bloodPressureSchema: SchemaDefinition = SchemaDefinition(url = "https://aiccelerate.eu/fhir/StructureDefinition/Ext-plt1-hsjd-blood-pressure", `type` = "blood-pressure-schema", id = "blood-pressure-schema", name = "blood-pressure-schema", rootDefinition = None, fieldDefinitions = None)
```

Comment:
Let's remove this variable and create two constants to keep its **url** and **name** in the corresponding test since these are the only fields that we use.